BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/CoreEventSource.cs

```diff
@@ -652,6 +652,9 @@ public void IngestionResponseTimeEventCounter(float responseDurationInMs)
         [Event(70, Message = "Updating Exception has failed. Error: {0}", Level = EventLevel.Error)]
         public void UpdateDataFailed(string error, string appDomainName = "Incorrect") => this.WriteEvent(70, error, this.nameProvider.Name);
 
+        [Event(71, Keywords = Keywords.UserActionable, Message = "TransmissionStatusEvent has failed. Error: {0}", Level = EventLevel.Error)]
```

Comment:
Need to tell the impact of this..
Be explicit - this will not affect actual transmission, only the notification is failing.